schema/playground/player_info.schema

```diff
@@ -0,0 +1,28 @@
+package playground;
+
+component PlayerInfo {
```

Comment:
What other info might this have? What is is used for? Ideally, when you see a component on an entity you should know what functionality is being added.

Reply:
Yup - just added this/PlayerControls to recreate the examples given on the qbi docs page:
https://docs.improbable.io/reference/13.4/shared/reference/query-based-interest#examples